src/ElectricPokemon.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon{
+    private static final String type = "electric";
+    List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
```

Comment:
Je schrijft je attacks hier met hoofdletters, maar in `PokemonGymImpl` word gecheckt op attacks zonder hoofdletter. Dat gaat wel goed, omdat er daar nog ee n`.toLowerCase()` overheen gaat, maar het is wel iets om je bewust van te zijn.